mobile/evidence/ios/Features/Tests/ProfileTests/ProfileTests.swift

```diff
@@ -10,89 +10,52 @@ import XCTest
 @testable import Models
 
 final class ProfileTests: XCTestCase {
-    
-//    func statusInputButtonTapped() {
-//        showModal = true
-//    }
  
     func testStatusInputButtonTrue() {
-        // Given - Arrange
         let model = StatusViewModel()
-        
-        // When - Act
-        let test = model.showModal
         model.statusInputButtonTapped()
-        
-        // Then - Assert
-        XCTAssertTrue(test)
+        XCTAssertTrue(model.showModal)
+    }
+    
+    func testModalCloseButtonFalse() {
+        let model = StatusViewModel()
+        model.modalCloseButtonTapped()
+        XCTAssertFalse(model.showModal)
+    }
+    
+    func testClearStatusInputButtonSuccess() {
+        let model = StatusViewModel()
+        model.statusInput = "Some input"
+        model.clearStatusInputButtonTapped()
+        XCTAssertTrue(model.statusInput.isEmpty)
+        XCTAssertFalse(model.showClearButton)
```

Comment:
Acho o uso do assert true e assert false um pouco menos legível do que o assert equal
```suggestion
        XCTAssertEqual(model.statusInput.isEmpty, true)
        XCTAssertEqual(model.showClearButton, false)
```